extension/src/Settings/Settings.ts

```diff
@@ -126,6 +127,10 @@ export interface IDRange {
   from: number;
   to: number;
 }
+export interface ISearchReplaceBeforeSaveXliff {
```

Comment:
Possibly missplaced interface depending on where the feature responsibility "belongs". As of this PRs current state my opinion is that this should reside within `src/Xliff`

Reply:
ofc. moved!